Demo/SerDes/SerDesDemo.cpp

```diff
@@ -0,0 +1,24 @@
+#include <iostream>
+#include <SerDes/SerDes.h>
+
+struct  ST {
+    int a;
+    float b;
+    char c;
+
+    ST(){}
+    ST(int a, float b, char c): a(a), b(b), c(c) {}
+};
+
+
+int main() {
+    ST obj(19,-145.3, 'g');
+    serialization("temp.txt",&obj);
+
+    ST another_obj;
+    deserialization("temp.txt",&another_obj);
+
+    std::cout << another_obj.a << std::endl;

```

Comment:
instead of printing check those two objects via `std::memcmp` or compare them componentwise and print the result.

Reply:
Add more unit tests with edge cases.